config/bolt/contenttypes.yaml

```diff
@@ -204,6 +204,34 @@ blockquotes:
     icon_many: "fa:quote-left"
     icon_one: "fa:quote-left"
     viewless: true
+
+showcases:
+    name: Showcases
+    singular_name: Showcase
+    fields:
+        title:
+            type: text
+            class: large
+        slug:
+            type: slug
+            uses: title
+        website_name:
+            type: text
+        website_link:
+            type: text
+            label: Link
+            placeholder: 'http://example.org/'
+        screenshot:
+            type: image
+        publication_date:
+            type: date
+            mode: datetime
+        contributor:
+            type: text
+        comment:
+            type: redactor
+    sort: -id

```

Comment:
Like discussed, let's create a field called `order` that we can use to order the entries. Otherwise, there's no way to determine the order in which they appear from the Editor.

Reply:
Done

Reply:
Nice, I have the feeling though it's not ordering on this yet, since it says `sort: -id` there? 🤔